examples/continue-statement/continue_statement.md

```diff
@@ -1,10 +1,10 @@
-# `Continue` statement
+# Continue statement
 
 `continue` statements can be used to skip the current iteration within a nearest enclosing `while` and `foreach` statement.
 
-::: continue_statement.bal :::
+::: code continue_statement.bal :::
 
-::: continue_statement.out :::
+::: out continue_statement.out :::
 
 ## Related links
 - [Break statement](/learn/by-example/break-statement/)
```

Comment:
@DamithaSenevirathne since we are talking about while and foreach statements, shall we point them also under the related links?

Reply:
They are already included in line 11 and 12